packages/legacy/core/App/components/misc/CredentialCard11.tsx

```diff
@@ -130,6 +130,8 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({
     primaryBodyContainer: {
       flexGrow: 1,
       padding,
+      marginLeft: -1 * logoHeight + padding,
+      marginRight: logoHeight * 2,
```

Comment:
This marginRight is taking up a lot of space and squeezing the credential attributes, does it need to be there? I checked out your branch and I tried removing it:
## With marginRight:
![2023-10-17_10-59](https://github.com/hyperledger/aries-mobile-agent-react-native/assets/36937407/804fae96-2a9a-44f1-ab8a-1afa2dc592d0)

## Without marginRight:
![2023-10-17_11-24](https://github.com/hyperledger/aries-mobile-agent-react-native/assets/36937407/9d885976-c06b-449b-bb52-9131073e4173)

Reply:
@wadeking98 

I updated my PR, thanks for checking. Apparently it's working now.

### **Before**
![Screenshot1](https://github.com/hyperledger/aries-mobile-agent-react-native/assets/97122568/3b71c6bc-b654-4986-831d-4294f7b148cb)

### **After**
![Screenshot2](https://github.com/hyperledger/aries-mobile-agent-react-native/assets/97122568/e12e485a-596b-4e70-a427-96d8d89b46c6)